rskj-core/src/main/java/co/rsk/peg/whitelist/constants/WhitelistConstants.java

```diff
@@ -0,0 +1,12 @@
+package co.rsk.peg.whitelist.constants;
+
+import co.rsk.peg.vote.AddressBasedAuthorizer;
+
+public class WhitelistConstants {
+
+    protected AddressBasedAuthorizer lockWhitelistChangeAuthorizer;
```

Comment:
What about making this field private, and providing a constructor where each sub-class call it using `super(AddressBasedAuthorizer)`

Reply:
I'm curious about this @apancorb. What advantage do you see in doing it that way?

Reply:
Since `WhitelistConstants` is declaring this instance variable it should also have the responsibility of initializing it. IMO, the advantage is that it provides better encapsulation. You can still get the reference by the getter present in the class, if needed by a sub-class.

Reply:
As discussed, let's keep it as it is for the time being. For consistency with the rest of the codebase. We can figure out a better way to access all the Constants later.

Reply:
I know that we are going to keep it as it is, however, I want to share my perspective on this case for future refactors.
IMO, if we apply a Strategy design pattern, we can create a private variable on the implementation, and give access only to the getter method through the interface. Definitely, as @apancorb said, it provides a better encapsulation.